### PR TITLE
Escrow unit tests coverage

### PR DESCRIPTION
**Pull request by vsavovlime**

### CODE REVIEW

File: contracts/Escrow_V2.sol [hunk 1]
```diff
@@ -88,5 +88,9 @@ contract Escrow_V2 {
         signers[_newSigner] = add;
     }
 
+    function editDappAdmin (address payable _dAppAdmin) public onlyDAppAdmin {
```
Comment (Daniel-K-Ivanov) [inline contracts/Escrow_V2.sol:91]: no need for `payable` address

File: test/escrow_v2-tests.js [hunk 2]
```diff
@@ -100,6 +100,35 @@ describe('Escrow Contract', function () {
...
         });
 
+        it('Should process 1000 fiat payments and refund correctly', async () => {
+            await tokenContract.contract.transfer(escrowContract.contractAddress, tokensToSend.mul(1010));
+            await deployer.wallet.sendTransaction({
+                to: escrowContract.contractAddress,
+                value: weiToSend.mul(1010)
+            });
+            
+            let signerBalanceBeforeFund = await deployer.provider.getBalance(deployer.wallet.address);
```
Comment (Daniel-K-Ivanov) [inline test/escrow_v2-tests.js]: This is not `signer`, it is `sender`
`signerBalanceBeforeFund` -> `senderBalanceBeforeFund`

File: test/escrow_v2-tests.js [hunk 3]
```diff
@@ -276,4 +339,28 @@ describe('Escrow Contract', function () {
...
+    describe('DAppAdmin functionality', () => {
+
+        beforeEach(async () => {
+            await initEscrowContract();
+
+            newDAppAdmin = ethers.Wallet.createRandom();
+            nonSigner.wallet = nonSigner.wallet.connect(deployer.provider);
+        });
+
+        it('Only dAppAdmin should be able to set a new dAppAdmin', async () => {
```
Comment (Daniel-K-Ivanov) [inline test/escrow_v2-tests.js]: Remove the `only` word. You are actually checking that `dAppAdmin should be able to set new dAppAdmin`

**Review by Daniel-K-Ivanov — CHANGES_REQUESTED** [review]

File: contracts/Escrow_V2.sol (hunk 1 above)
Comment (vsavovlime) [reply contracts/Escrow_V2.sol:91]: line 15 -> address payable public dAppAdmin; dAppAdmin is payable, so payable can't be removed

File: test/escrow_v2-tests.js (hunk 2 above)
Comment (vsavovlime) [reply test/escrow_v2-tests.js]: Done

File: test/escrow_v2-tests.js (hunk 3 above)
Comment (vsavovlime) [reply test/escrow_v2-tests.js]: Done

**Review by Daniel-K-Ivanov — APPROVED** [review]

**Pull request merged**